.github/actions/docker-build-and-push-cuda/action.yaml

```diff
@@ -0,0 +1,121 @@
+        restore-keys: |
+          ccache-${{ inputs.platform }}-
+
+    - name: Restore apt-get
+      uses: actions/cache/restore@v4
+      with:
+        path: |
+          var-cache-apt
+        key: apt-get-${{ inputs.platform }}-${{ hashFiles('src/**/package.xml') }}
+        restore-keys: |
+          apt-get-${{ inputs.platform }}-
+
+    - name: Inject cache into docker
+      uses: reproducible-containers/buildkit-cache-dance@v3.1.2
+      with:
+        cache-map: |
+          {
+            "root-ccache": "/root/.ccache",
+            "var-cache-apt": "/var/cache/apt"
+          }
+        skip-extraction: true
+
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - name: Docker meta for autoware:universe-sensing-perception-devel-cuda
+      id: meta-universe-sensing-perception-devel-cuda
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        tags: |
+          type=raw,value=universe-sensing-perception-devel-cuda-${{ inputs.platform }}
+          type=raw,value=universe-sensing-perception-devel-cuda-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+        bake-target: docker-metadata-action-universe-sensing-perception-devel-cuda
+        flavor: |
+          latest=false
+
+    - name: Docker meta for autoware:universe-sensing-perception-cuda
+      id: meta-universe-sensing-perception-cuda
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        tags: |
+          type=raw,value=universe-sensing-perception-cuda-${{ inputs.platform }}
+          type=raw,value=universe-sensing-perception-cuda-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+        bake-target: docker-metadata-action-universe-sensing-perception-cuda
+        flavor: |
+          latest=false
+
+    - name: Docker meta for autoware:universe-devel-cuda
+      id: meta-universe-devel-cuda
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        tags: |
+          type=raw,value=universe-devel-cuda-${{ inputs.platform }}
+          type=raw,value=universe-devel-cuda-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+        bake-target: docker-metadata-action-universe-devel-cuda
+        flavor: |
+          latest=false
+
+    - name: Docker meta for autoware:universe-cuda
+      id: meta-universe-cuda
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        tags: |
+          type=raw,value=universe-cuda-${{ inputs.platform }}
+          type=raw,value=universe-cuda-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+        bake-target: docker-metadata-action-universe-cuda
+        flavor: |
+          latest=auto
```

Comment:
[question] do you intend to update the same metadata for bake-target on the same image?

Reply:
The tag names are different from all images.